docs/tutorials/backends/t2star_backend.py

```diff
@@ -0,0 +1,128 @@
+        self._initial_prob_plus = initial_prob_plus
+        self._readout0to1 = readout0to1
+        self._readout1to0 = readout1to0
+        self._dt_factor = dt_factor
+        super().__init__(configuration)
+
+    # pylint: disable = arguments-differ
+    def run(self, qobj):
+        """
+        Run the T2star backend
+        """
+        shots = qobj.config.shots
+        result = {
+            "backend_name": "T2star backend",
+            "backend_version": "0",
+            "qobj_id": 0,
+            "job_id": 0,
+            "success": True,
+            "results": [],
+        }
+
+        for circ in qobj.experiments:
+            nqubits = circ.config.n_qubits
+            counts = dict()
+            if self._readout0to1 is None:
+                ro01 = np.zeros(nqubits)
+            else:
+                ro01 = self._readout0to1
+            if self._readout1to0 is None:
+                ro10 = np.zeros(nqubits)
+            else:
+                ro10 = self._readout1to0
+            for _ in range(shots):
+                if self._initial_prob_plus is None:
+                    prob_plus = np.ones(nqubits)
+                else:
+                    prob_plus = self._initial_prob_plus.copy()
+
+                clbits = np.zeros(circ.config.memory_slots, dtype=int)
+                for op in circ.instructions:
+                    qubit = op.qubits[0]
+
+                    if op.name == "delay":
+                        delay = op.params[0]
+                        t2star = self._t2star[qubit] * self._dt_factor
+                        freq = self._f_guess[qubit] / self._dt_factor
+
+                        prob_plus[qubit] = (
+                            self._a_guess[qubit]
+                            * np.exp(-delay / t2star)
+                            * np.cos(2 * np.pi * freq * delay + self._phi_guess[qubit])
+                            + self._b_guess[qubit]
+                        )
+
+                    if op.name == "measure":
+                        # we measure in |+> basis which is the same as measuring |0>
+                        meas_res = np.random.binomial(
+                            1,
+                            (1 - prob_plus[qubit]) * (1 - ro10[qubit])
+                            + prob_plus[qubit] * ro01[qubit],
+                        )
+                        clbits[op.memory[0]] = meas_res
+
+                clstr = ""
+                for clbit in clbits[::-1]:
+                    clstr = clstr + str(clbit)
+
+                if clstr in counts:
+                    counts[clstr] += 1
+                else:
+                    counts[clstr] = 1
+            result["results"].append(
+                {
+                    "shots": shots,
+                    "success": True,
+                    "header": {"metadata": circ.header.metadata},
+                    "data": {"counts": counts},
+                }
+            )
+        return Result.from_dict(result)
```

Comment:
In the main branch I think it was replaced by a `MockJob` (to be probably replaced again by `FakeJob` by #115, to be replaced by Ethan's).

Reply:
Yes, I just realized I didn't update this in the tutorial.

Reply:
Done, along with several other updates that were missing.